src/NexusMods.Abstractions.Downloads/IDownloadsService.cs

```diff
@@ -57,11 +62,21 @@ public interface IDownloadsService
     /// </summary>
     void PauseAll();
     
+    /// <summary>
+    /// Pauses all active downloads for a specific game.
+    /// </summary>
+    void PauseAllForGame(GameId gameId);
+    
     /// <summary>
     /// Resumes all paused downloads.
     /// </summary>
     void ResumeAll();
     
+    /// <summary>
+    /// Resumes all paused downloads for a specific game.
+    /// </summary>
+    void ResumeAllForGame(GameId gameId);
+    
```

Comment:
You could take an `Optional<GameId>` in these methods and condense the API a bit